### ISSUE TITLE
Unable to hide wordpress logo and visit site 

### ISSUE THREAD

**Issue by ANANPATW000** [bueltge/Adminimize#62]
Hello, Post all updates 1.10.6 of adminimize plugin, the "wordpress logo with full menu" and "visit site"  on admin bar front end option is still visible even after tick to hide. All versions after 1.10.6 has no effect of tick/untick of above properties and still visible on top left of wordpress site. Please can you check and advise. Thanks

**Comment by bueltge** [comment]
Have you try the current release 1.11.2, this version fixed also this topic?

**Comment by ANANPATW000** [comment]
I tried it but still same issue

**Comment by bueltge** [comment]
Do you have really active this settings for front end and/or back end? It is different to set options. Have you the problem also on front end and back end or is different?

**Comment by ANANPATW000** [comment]
Yes. I set it to hide at both front and back end. But WP logo with menus
still displays at front end and "visit store" submenu visible at back end.
